<ui version="4.0" >
 <class>WidgetRgbT</class>
 <widget class="QWidget" name="WidgetRgbT" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="4" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Selection tools:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QToolButton" name="bBrush" >
     <property name="toolTip" >
      <string>Brush for mesh refinement</string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="editrgbtri.qrc" >:/images/brush.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QToolButton" name="bEraser" >
     <property name="toolTip" >
      <string>Eraser for mesh coarsening</string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="editrgbtri.qrc" >:/images/eraser.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QToolButton" name="bSelectionSingle" >
     <property name="toolTip" >
      <string>Single face selector</string>
     </property>
     <property name="text" >
      <string>Single Selection</string>
     </property>
     <property name="icon" >
      <iconset resource="editrgbtri.qrc" >:/images/select_face.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <property name="usesScrollButtons" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Interactive</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Brush Radius (pixel)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>119</width>
             <height>23</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="sRadius" >
           <property name="layoutDirection" >
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum" >
            <number>10</number>
           </property>
           <property name="maximum" >
            <number>500</number>
           </property>
           <property name="value" >
            <number>30</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QCheckBox" name="cLevelEnabled" >
           <property name="text" >
            <string>Refinement Level</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>42</width>
             <height>23</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="sBrushLevel" >
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum" >
            <number>8</number>
           </property>
           <property name="value" >
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QCheckBox" name="cLengthEnabled" >
           <property name="text" >
            <string>Edge Length</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>41</width>
             <height>23</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="sBrushLength" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly" >
            <bool>false</bool>
           </property>
           <property name="decimals" >
            <number>6</number>
           </property>
           <property name="maximum" >
            <double>9999.989999999999782</double>
           </property>
           <property name="value" >
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>202</width>
           <height>71</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Primitives</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Edge Split</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>131</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="bEdgeSplit" >
           <property name="toolTip" >
            <string>Execute a split on a green edge. Two adjacent triangles must be selected.</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="editrgbtri.qrc" >:/images/edgesplit.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Vertex Removal</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>81</width>
             <height>41</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="bEdgeCollapse" >
           <property name="toolTip" >
            <string>Execute a vertex removal. Three triangle sharing a vertex must be selected.</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="editrgbtri.qrc" >:/images/edgecollapse.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>51</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>bSelectionSingle</tabstop>
 </tabstops>
 <resources>
  <include location="editrgbtri.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>cLevelEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>sBrushLevel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>78</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel" >
     <x>275</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cLengthEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>sBrushLength</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>82</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel" >
     <x>276</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
